### DISCUSSION TITLE
Carte Google Maps avec Service Google My Maps

### DISCUSSION THREAD

**Discussion by e-stephanie** [AmauriC/tarteaucitron.js#857] [Q&A]
Bonjour, 

Je suis débutante, c'est mon premier site, donc je tâtonne un peu... 

J'ai intégrer sur mon site une carte Google My Maps qui possède l'iframe suivant :
<iframe src="https://www.google.com/maps/d/embed?mid=1kkisRYOPPGgZ3DmRFwxXTUK-6HKB1vye&ehbc=2E312F" width="100%" height="100%"></iframe>

Sur le guide d'installation je n'ai pas trouvé **Google My Maps** au niveau des services, j'ai utilisé **Google Maps (no api)**, j'ai suivi les directives pensant que ça fonctionnerait mais ce n'est pas le cas, j'ai la carte du monde qui s'affiche et non pas ma carte personnalisée. 
Je n'arrive pas à trouver de solution pour remédier à cela, comment faire ?

En vous remerciant!

**Comment by AmauriC** [comment]
👋  
Il ne sera pas possible d'intégrer cette carte avec les services Gmaps existants qui ne prennent pas en charge ce type de cartes.

Vous pouvez le faire avec le service Gmaps classique et une fonction js en callback pour recréer la zone mais c'est plus compliqué 😬

**Reply by e-stephanie** [reply]
D'accord, merci beaucoup pour votre réponse, je vais faire autrement alors ;).

**Comment by juluz75** [comment]
Bon, après, il est indiqué en mode auto que les cartes sous Google My Maps, ne génèrent pas de cookie ... donc ...

**Reply by AmauriC** [reply]
Il n'y a pas que les cookies, à partir du moment où un site charge du contenu tiers (de chez Google ici), il y a transfert de données.

**Reply by juluz75** [reply]
OK Amauri - alors ces cartes Google My Maps devraient être prises en charge sans un callback complexe à mettre en place.
